_persistence/exec/readme.md

```diff
@@ -5,8 +5,8 @@ title: Exec - Persistence
 type: persistence
 description: "This service allows you to execute commands in the underlying OS to persist item states. It could be used to, for example, call the original `rrdtool` CLI tool."
 source: https://github.com/openhab/openhab1-addons/blob/master/bundles/persistence/org.openhab.persistence.exec/README.md
-since: 1x
-install: manual
+since: 2x
+install: auto
```

Comment:
I see. I'd follow your ruling and ignore this bug for the sake of merging the transformations.

Would you please open a short issue report? The previous message would be more than enough. Thanks!